f2/src/__stories__/Start.stories.mdx

```diff
@@ -1,7 +1,90 @@
-import { Meta } from '@storybook/addon-docs/blocks'
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
+import ThemeDecorator from '../../.storybook/themeDecorator'
+import { Code } from '@chakra-ui/core'
 
-<Meta title="1- Start" />
+<Meta id="1-start" title="Getting started" decorators={[ThemeDecorator]} />
 
-# Start
+# Getting started
 
-This documentation is meant to help build and modify pages.
+This documentation is meant to help build and modify pages and forms.
+
+---
+
+# Reading Stories
+
+You can access any documented component from the sidebar. Storybook is consisted of multiple `Stories`, which each correspond to a component or one of its variations.
+
+Each Story also lets you view a code preview showing how to use the component in code. You can copy it from there as well as a properties table when applicable. Shows property name, possible values and types, and default values when applicable.
+
+## Component variations
+
+If a component is missing, it has not been documented. You can create a Github issue to request that someone document the missing component, or you can create the
```

Comment:
Don't leave me hanging! :P

Reply:
oops! I'm on it, basically, this is for the